### PR TITLE
Adds a 3rd Display Ad to Standard Articles

### PR DESCRIPTION
**Pull request by ashleyjelks**
Adds a third ad placement in Standard Articles.

https://artsyproduct.atlassian.net/browse/FX-1839
https://artsyproduct.atlassian.net/browse/FX-1837

Kapture 2020-03-19 at 13 27 41

Kapture 2020-03-19 at 17 05 22

### CODE REVIEW

File: src/Components/Publishing/Sections/Sections.tsx [hunk 1]
```diff
@@ -290,29 +331,47 @@ export class Sections extends Component<Props, State> {
...
-            : AdDimension.Mobile_Feature_InContentLeaderboard1
-          : AdDimension.Desktop_NewsLanding_Leaderboard1
-
+        const adDimension = this.getAdDimension(isStandardArticle)
         const marginTop = this.getAdMarginTop(section)
 
         firstAdInjected = true
         indexAtFirstAd = indexAtFirstAd === null ? index : indexAtFirstAd // only set this value once; after the index where 1st ad injection is found
 
+        console.log(
```
Comment (sweir27) [inline src/Components/Publishing/Sections/Sections.tsx]: 🙈

**Comment by ashleyjelks** [comment]
#mergeongreen

**Comment by sweir27** [comment]
The tag didn't seem to work-- going to merge this!

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.33.2` :rocket:

**Pull request merged**